### PR TITLE
feat(Icon): add success, failure icons

### PR DESCRIPTION
**Pull request by susheelachoudry**

### CODE REVIEW

File: src/icons/SuccessSnackbar/SuccessSnackbar.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getThemeColors } from '../../_helpers/theme';
+import colors from '../../tokens/colors';
+
+function Test({ width, height, fill }) {
```
Comment (dhruvdutt) [inline src/icons/SuccessSnackbar/SuccessSnackbar.js]: Fix this.

File: src/icons/index.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ const icons = {
   search: require('./Search').default,
   smartCollect: require('./SmartCollect').default,
   subscription: require('./Subscription').default,
+  successSnackbar: require('./SuccessSnackbar').default,
```
Comment (dhruvdutt) [inline src/icons/index.js:26]: Icons are not specific to features. Why snackbar in name?

Comment (dhruvdutt) [reply src/icons/index.js:26]: Also, add the other icon for failure.

File: src/icons/SuccessSnackbar/SuccessSnackbar.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getThemeColors } from '../../_helpers/theme';
+import colors from '../../tokens/colors';
+
+function SuccessSnackbar({ width, height, fill }) {
```
Comment (dhruvdutt) [inline src/icons/SuccessSnackbar/SuccessSnackbar.js:6]: Add in native as well. Look at other implementations.

File: src/icons/SuccessSnackbar/SuccessSnackbar.js (hunk 1 above)
Comment (susheelachoudry) [reply src/icons/SuccessSnackbar/SuccessSnackbar.js]: it was by mistake. rectified immediately.

File: src/icons/index.js (hunk 2 above)
Comment (susheelachoudry) [reply src/icons/index.js:26]: Sure.

File: src/icons/Failure/Faliure.native.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```
Comment (dhruvdutt) [inline src/icons/Failure/Faliure.native.js:1]: Typo in filename.

**Comment by susheelachoudry** [comment]
@kamleshchandnani Could you review and approve please

**Comment by kamleshchandnani** [comment]
@susheelachoudry did you pass your svg through this? https://react-svgr.com/playground/ for web and native? it removes few of the unnecessary things and add few missing pieces to svg

**Comment by susheelachoudry** [comment]
@kamleshchandnani I wasn't aware of it. Now I have updated the PR with the changes. please check.

**Pull request merged**